main.tf

```diff
@@ -1,3 +1,14 @@
+provider "aws" {
```

Comment:
Why do you need an extra provider for the dkim resources?

Reply:
SES can only be deployed in a few regions, so it's highly likely that this will need to be changed when using this module

Reply:
true, but that'll be the case for all the resources in the module, not just `aws_ses_domain_identity` and `aws_ses_domain_dkim`, no?

Also, you can override the provider for the whole module externally: https://www.terraform.io/docs/modules/usage.html#passing-providers-explicitly

Reply:
Ah yep, I see what I've done here now 😑 

I was trying to ensure only the SES resources would launch in the few regions available, but still allow the other resources to launch in the default region. However, Route53 records are Global, so it doesn't matter in this case 👍 ... And right now I can't think of other resources that'd be needed for this module that the user would want to keep in the default region

(eg. If Route53 resources were region specific, you wouldn't be able to use a zone id not in the region specified for SES)

I'll modify the PR 👍

Reply:
Ok, thanks for the clarification, I now understand why you added that 🙂 
And thanks for taking the time to modify the PR